tools/SourceKit/tools/sourcekitd/lib/API/sourcekitdAPI-InProc.cpp

```diff
@@ -11,8 +11,8 @@
 //===----------------------------------------------------------------------===//
 
 #include "DictionaryKeys.h"
-#include "sourcekitd/sourcekitd.h"
 #include "sourcekitd/Internal.h"
+#include "sourcekitd/sourcekitd.h"
```

Comment:
`sourcekitd/Internal.h` also includes `sourcekitd/sourcekitd.h` so you could remove the include of `sourcekitd/sourcekitd.h` from here completely.